diagnostic.md

```diff
@@ -177,5 +196,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+Since the screen is wider than 400px, but less than 800px, the elements will
+be 30px wide. 
```

Comment:
No. 550px is greater than 400px, but less than 800px. That means it meets the requirement for min-width 400px, so the width if 550px times 25%, which is 137.5px.